### PR TITLE
bare-metal-os module cleanup

### PR DESCRIPTION
**Pull request by adean20csc**
This PR is mostly comments to match documentation.

I did remove the create_inventory stuff since we have changed that.

### CODE REVIEW

**Comment by mtnbikenc** [comment]
The create_inventory stuff was not completely cleaned up.  Some vars and files still exist that reference stuff.  Also, with some other variables changed it would make sense to test this to make sure there are no errors if defaults are left blank.

File: modules/bare-metal-os/inventory/group_vars/bare_metal_os.yml [hunk 1]
```diff
@@ -1,42 +1,31 @@
...
-#   bare-metal-os/site_deploy (ifcfg template)
+# The IP address of the DNS server, used in the 'dhcp-server' role 'ifcfg' template.
 dns1: x.x.x.x
 
-# Usage:
-#   bare-metal-os/site_inventory
-inventory_path: ./inventory/
-inventory_csv_file: ./inventory/example_inventory.csv
+# The path to the DCAF inventory.
+inventory_path: ../../inventory/
```
Comment (mtnbikenc) [inline modules/bare-metal-os/inventory/group_vars/bare_metal_os.yml]: This var should be removed as well.

File: modules/bare-metal-os/inventory/host_vars/example_host.yml [hunk 2]
```diff
@@ -1,57 +1,71 @@
 ---
-# The ip and pxe ip address of the host
-ansible_ssh_host: 192.168.0.249
-pxe_ip_address: 192.17.16.249
+# The ip and pxe ip address of the host.
+ansible_ssh_host: x.x.x.x
```
Comment (mtnbikenc) [inline modules/bare-metal-os/inventory/host_vars/example_host.yml]: This var is not required.

**Comment by mtnbikenc** [comment]
I've tested this PR with defaults from building autodeploynode to bare-metal-os deployment in a VM.
Review 👍

**Comment by jcpowermac** [comment]
Review 👍

**Pull request merged**